app/controllers/insured/verification_documents_controller.rb

```diff
@@ -3,10 +3,14 @@
 class Insured::VerificationDocumentsController < ApplicationController
   include ApplicationHelper
 
-  before_action :get_family
-  before_action :updateable?, :find_type, :find_docs_owner, only: [:upload]
+  before_action :set_current_person
+  before_action :find_type, :find_docs_owner, only: [:upload]
+  before_action :check_for_consumer_role
 
   def upload
+    consumer_role = person_consumer_role
+    authorize consumer_role, :verification_document_upload?
```

Comment:
@saipraveen18 You are already assigning `@person.consumer_role` to `@consumer_role`. You can use it directly like the one below.

```suggestion
    authorize @consumer_role, :verification_document_upload?
```